web-client/public/js/upload.js

```diff
@@ -135,43 +135,17 @@ export const upload = function (sliderObject, sliderGroupController, drawGraph,
         delay: { show: 700, hide: 100 }
     });
 
-    // Normalization Controller
-    var MIN_EDGE_WEIGHT_NORMALIZATION = 0.0001;
-    var MAX_EDGE_WEIGHT_NORMALIZATION = 1000;
-
-    var valueValidator = function (min, max, value) {
-        return Math.min(max, Math.max(min, value));
-    };
-
-    var edgeWeightNormalizationInputValidation = function (value) {
-        return value === "" ? "" : valueValidator(MIN_EDGE_WEIGHT_NORMALIZATION, MAX_EDGE_WEIGHT_NORMALIZATION, value);
-    };
-
-    var synchronizeNormalizationValues = function (value) {
-        var validated = edgeWeightNormalizationInputValidation(value);
-        $("#normalization-max").val(validated);
-        $("#edge-weight-normalization-factor-menu").val(validated);
-        drawGraph(uploadState.currentNetwork, sliders, nodeColoring);
-    };
-
-    $("#normalization-button").click(function () {
-        synchronizeNormalizationValues($("#normalization-max").val());
-    });
-
-    $("#reset-normalization-factor-menu, #resetNormalizationButton").click(function () {
-        synchronizeNormalizationValues("");
-    });
-
-    $("#edge-weight-normalization-factor-menu").on("change", function () {
-        synchronizeNormalizationValues($("#edge-weight-normalization-factor-menu").val());
-    });
-
     var GREY_EDGE_THRESHOLD_MENU = "#gray-edge-threshold-menu";
     var GREY_EDGE_THRESHOLD_SLIDER_SIDEBAR = "#grayThresholdInput";
     var GREY_EDGE_THRESHOLD_TEXT_SIDEBAR = "#grayThresholdValue";
 
     // Gray Edge Controller
 
+    let valueValidator = (min, max, value) => {
+        return Math.min(max, Math.max(min, value));
+    };
+
```

Comment:
No, there shouldn't be any additions to _upload.js_ unless they are transitional. I think you forgot to take this out.

Reply:
This was needed for gray edge threshold, which is still included in upload.js, when I took it out gray edge broke, so I had to add it

Reply:
OK. So it can go away when gray edge threshold gets moved. Mark this with a TODO comment then, so we don't forget about it later.